modules/ducktests/tests/ignitetest/services/zk/zookeeper.py

```diff
@@ -27,8 +27,9 @@ class ZookeeperSettings:
     """
     Settings for zookeeper quorum nodes.
     """
-    def __init__(self, tick_time=1000, init_limit=10, sync_limit=5, client_port=2181):
-        self.tick_time = tick_time
+    def __init__(self, min_session_timeout=2000, init_limit=10, sync_limit=5, client_port=2181):
+        self.tick_time = min_session_timeout // 2
```

Comment:
I suppose that it's better leave tick time and add assertion that min_session_timeout is at least twice as tick_time.
It's ok to pass tickTime as 1000 ms, and set minSessionTimeout as 3000 ms

By default, let them be 1000 and 2000

Reply:
Fixed